scripts/data_ingest/main.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import logging.config
+import os
+import sys
+from http import HTTPStatus
+from pathlib import Path
+from pprint import pprint
+
+import requests
+
+ADMIN_EMAIL_ENV = "SUPERUSER_EMAIL"
+ADMIN_PASSWORD_ENV = "SUPERUSER_PASSWORD"
+ADMIN_TOKEN_ENV = "SUPERUSER_TOKEN"
+BULK_IMPORT_ENDPOINT = "/api/v1/bulk-imports/cclw/law-policy"
+
+
+DEFAULT_LOGGING = {
```

Comment:
This logging configuration is different from the config we are using in the ingest stage which was an example / baseline of the logging format using json-logging just checking we're okay with that? I guess it doesn't really matter at all as we aren't querying these logs but just checking. 

See this below and here:
https://github.com/climatepolicyradar/navigator-data-ingest/blob/main/src/navigator_data_ingest/main.py#L30

Reply:
yeah, these are really just info logs for a local script, I think the default format is easier to read (these log lines are aimed at human admins)